pkg/vegeta/report.go

```diff
@@ -19,11 +21,50 @@ const (
 	// TextFormat typedef for query param "text"
 	TextFormat Format = "text"
 	// HistogramFormat typedef for query param "histogram"
-	//HistogramFormat Format = "histogram"
+	HistogramFormat Format = "histogram"
 	// BinaryFormat typedef for query param "binary"
 	BinaryFormat Format = "binary"
+	// DefaultBucketString Default Bucket String
+	DefaultBucketString string = "0,500ms,1s,1.5s,2s,2.5s,3s"
 )
 
+// SplitFormat splits the Format into sub-Formats separated by '-' using StringsToFormat
+func (f Format) SplitFormat() (fs []Format) {
+	sf := string(f)
+	sli := strings.Split(sf, "-")
+	for _, s := range sli {
+		fs = append(fs, Format(s))
+	}
+	return
+}
+
+// GetFormat returns the actual Format string. It is useful in case of Clubbed Format string
```

Comment:
What are these helpers all about ? Why would we need a Clubbed Format string ?

Not sure I follow these helpers to get the Format ? Was it done only from a testing perspective ?

I dont see it being used in the endpoints package, other than the Bucket and GetFormat()